pantomime-lib/Framework/Pantomime/CWParser.m

```diff
@@ -301,6 +301,16 @@ + (void) parseContentType: (NSData *) theLine
         [thePart setContentType: [[aData asciiString] lowercaseString]];
     }
 
+    //
+    // We decode our protocol (if we need to)
+    //
+    aRange = shrinkRange([theLine rangeOfCString: "protocol="  options: NSCaseInsensitiveSearch]);
+
+    if (aRange.location != NSNotFound)
+    {
+        [thePart setProtocol: [CWParser _parameterValueUsingLine: theLine  range: aRange  decode: NO  charset: nil]];
```

Comment:
Bonus points for using the built-in protocol. I didn't even know (remember?) it existed.

Reply:
👍